Dangerfile

```diff
@@ -17,3 +17,7 @@ end
 if pr_body.length < 5
   fail "Please provide a changelog summary in the Pull Request description @#{pr_author}"
 end
+
+unless files_modified.any? { |a| a.include?("CHANGELOG.md") }
```

Comment:
I'd opt for just dealing with one level of allowing "docs/thingy"  - I'd imagine some special casing  around `file_modified["docs/"]` working if any file inside that dir is changes.

Other options is to support whatever glob thing CocoaPods supports ( I think it uses `Dir.glob` under the hood )